components/xsd-fu/templates-java/OMEXMLModelEnumHandler.template

```diff
@@ -91,10 +91,15 @@ public class ${klass.langType}EnumHandler implements IEnumerationHandler {
 {% for value in klass.possibleValues %}\
     p.put("^\\\s*${value}\\\s*", "${value}");
 {% end %}\
-{% if klass.langType in config %}\
     // BEGIN custom enumeration mappings
```

Comment:
The conditional ends up being a bit more complex than that--this is always true--so I redid it to make the custom mapping line comments more informative which should hopefully have the same effect.